cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/objects-side-bar.tsx

```diff
@@ -86,6 +90,9 @@ function ObjectsSideBar(props: Props): JSX.Element {
             collapsible
             trigger={null}
             collapsed={sidebarCollapsed}
+            onTransitionEnd={() => {
+                canvasInstance.fit();
+            }}
```

Comment:
Looks like fit works very often. Even when we don't collapse it.

Reply:
@bsekachev, what do you mean, can you record a case? I've only noticed that transition not always triggered in exact time to accurate fitting.